jsonify.js

```diff
@@ -1,5 +1,6 @@
 function clean (node) {
-  if (node.source) delete node.source.input
```

Comment:
Let’s keep this line for compatibility issues (I am using these tests in different parsers).